tests/wpt/mozilla/tests/mozilla/bluetooth/getPrimaryServices/get-different-services-after-reconnection-with-uuid.html

```diff
@@ -1,33 +0,0 @@
-<!doctype html>
-<meta charset="utf-8">
-<script src="/resources/testharness.js"></script>
-<script src="/resources/testharnessreport.js"></script>
-<script src="/_mozilla/mozilla/bluetooth/bluetooth-helpers.js"></script>
-<script>
-'use strict';
-promise_test(t => {
-    window.testRunner.setBluetoothMockDataSet(adapter_type.two_heart_rate);
-    return window.navigator.bluetooth.requestDevice({
-        filters: [{services: [heart_rate.name]}]
-    })
-    .then(device => device.gatt.connect())
-    .then(gattServer => {
-        let services1;
-        return gattServer.getPrimaryServices(heart_rate.name)
-        .then(services => services1 = services)
-        .then(() => gattServer.disconnect())
-        .then(() => gattServer.connect())
-        .then(() => gattServer.getPrimaryServices(heart_rate.name))
```

Comment:
My comment wasn't intended to suggest that we should delete the `with-uuid` variants of the tests. Is the test coverage meaningful? Do they exist in google's test suite, or are these tests ones that you wrote without a reference?